api/entity_config/config.go

```diff
@@ -12,6 +12,12 @@ func ConfigValue(network *net.Network, application, entity, config string) (stri
 	if nil != err {
 		return "", err
 	}
+
+	var value string;
+	err = json.Unmarshal(bytes, &value)
+	if nil == err {
```

Comment:
Then I would suggest unmarshalling to type `interface{}` rather than `string` and let the client code examine it with a switch on its type. (cf. http://blog.golang.org/json-and-go)